src/lib/events.ts

```diff
@@ -407,3 +407,49 @@ export function without<T extends CollectionEntry<"events">>(
 ) {
   return events.filter((event) => event.data.status !== status);
 }
+
+export async function getPastEventsWithVodTalks() {
+  const events = (await getEventsCollection()).sort(
+    (a, b) =>
+      dayjs(a.data.date).diff(b.data.date) ||
+      (b.data.type === "event" ? 1 : 0) - (a.data.type === "event" ? 1 : 0),
+  );
+
+  const talks = await getCollection("talks", (event) => event.data.vod);
+  const eventWithTalks = events.map((event) => {
```

Comment:
```suggestion
  const eventWithTalksWithVods = events.map((event) => {
```
I think the name is not very clear